machine_data_model/nodes/connectors/opcua_connector.py

```
@@ -0,0 +1,440 @@
+            parent_path = "/".join(path_parts[:-1])
+            method_id = path_parts[-1]
+            parent_node = await self._async_get_remote_node(parent_path)
+            result = await parent_node.call_method(method_id, *params)
+        except UaError as exp:
+            _logger.error(exp)
+        return result
+
+    def __str__(self) -> str:
+        return (
+            "OpcuaConnector("
+            f"name={repr(self.name)}, "
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)